tests/checkpoint/helpers.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2024 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any, Dict
+
```

Comment:
should this be in `common`?

Reply:
they could. I made them specifically for checkpointing tests, but if you think they would be useful in common. However, if putting them in common requires a more time-consuming review then maybe not

Reply:
I think it's fine for now... though it seems nice to unify eventually.